pages/cloud/lake-effect.md

```diff
@@ -1,47 +1,71 @@
 ## LakeEffect Research Cloud _(On-Premise Cloud)_
 
 
-![](LakeEffectLogo_2.jpg)
+![](../images/cloud/LakeEffectLogo_2.jpg)
 
-CCR hosts a subscription based on-premise cloud service affectionately known as **"Lake Effect"**, which provides access to virtual servers, storage and networking that users setup and manage themselves. It is what is referred to as Infrastructure as a Service and is equivalent in functionality to Amazon's EC2 Cloud service.
+CCR hosts a subscription based on-premise cloud service affectionately known as **"LakeEffect"**, which provides access to virtual servers, storage and networking that users setup and manage themselves. It is what is referred to as Infrastructure as a Service and is equivalent in functionality to Amazon's EC2 Cloud service.
 
-Although high performance computing (HPC) is our largest and most widely used service, CCR recognizes that not all research requires number crunching on large scale computers (clusters). Lake Effect, was put into service to provide flexible infrastructure to support research computing such as webservers, databases, and long-running services that are not suited for HPC clusters. This along with our HPC platform allows us to provide a complete set of tools to support all the different research groups and collaborations througout UB.
+Although high performance computing (HPC) is our largest and most widely used service, CCR recognizes that not all research requires number crunching on large scale computers (clusters). LakeEffect, was put into service to provide flexible infrastructure to support research computing such as webservers, databases, and long-running services that are not suited for HPC clusters. This along with our HPC platform allows us to provide a complete set of tools to support all the different research groups and collaborations througout UB.
```

Comment:
throughout is misspelled in the last sentence.  Sorry I missed that before

Reply:
fixed.